assets/js/components/notifications/UnsatisfiedScopesAlert.js

```diff
@@ -202,14 +244,26 @@ export default function UnsatisfiedScopesAlert( { id, Notification } ) {
 				title={ title }
 				description={ <Description text={ message } /> }
 				actions={
-					<CTALink
-						id={ id }
-						ctaLabel={ ctaLabel }
-						ctaLink={ connectURL }
-						onCTAClick={ () => {
-							doingCTARef.current = true;
-						} }
-					/>
+					inProgressModuleSetup ? (
+						<CTALink
+							id={ id }
+							ctaLabel={ sprintf(
+								/* translators: %s: Module name */
+								__( 'Redo %s setup', 'google-site-kit' ),
```

Comment:
Hey @nfmohit, although this meets the IB, it does deviate from the AC and while on the face of it, it's a logical change, I've noticed a couple of edge cases where it doesn't work so well.

For one thing, the current wording when a module requests scopes for more than one service is a bit unclear. Here's how it looks for Analytics which also requests a Tag Manager scope:

![image](https://github.com/user-attachments/assets/0004c05d-acf7-4729-aafe-96a97be3831a)

This is compounded if you choose to connect a different module rather than redoing the last one, for example here I've tried to setup AdSense instead of redoing the Analytics setup:

![image](https://github.com/user-attachments/assets/82f083af-d1a4-4b46-a197-ec0c54da1417)

https://github.com/user-attachments/assets/5df7a125-af8c-4b77-9731-a11fc4f7bb01

There's more than one way we could address this but I think keeping the button label as "Redo setup" would be the simplest way forward in the short term, it will simply redo the last setup action the user tried which makes sense and avoids the confusing copy combinations. We could potentially iterate on it to further improve the copy in a subsequent issue. WDYT?

Reply:
I agree. I didn't consider this scenario previously but thank you for raising this. I've reverted this change. Thank you!

Reply:
Top stuff, thanks Nahid!